package.json

```diff
@@ -43,33 +43,35 @@
     "npm": ">=3.0.0"
   },
   "devDependencies": {
-    "aegir": "^21.4.5",
+    "aegir": "^21.8.1",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1",
-    "lodash": "^4.17.11",
+    "just-range": "^2.1.0",
     "memdown": "^5.1.0",
     "multihashes": "~0.4.15",
     "multihashing-async": "~0.8.0",
     "ncp": "^2.0.0",
     "rimraf": "^3.0.0",
-    "sinon": "^9.0.1"
+    "sinon": "^9.0.2"
   },
   "dependencies": {
-    "base32.js": "~0.1.0",
     "bignumber.js": "^9.0.0",
+    "buffer": "^5.5.0",
     "bytes": "^3.1.0",
-    "cids": "^0.8.0",
-    "datastore-core": "~0.7.0",
+    "cids": "~0.8.0",
+    "datastore-core": "^1.0.0",
     "datastore-fs": "~0.9.0",
+    "datastore-idb": "^1.0.0",
     "datastore-level": "~0.14.0",
     "debug": "^4.1.0",
     "err-code": "^2.0.0",
-    "interface-datastore": "^0.8.0",
-    "ipfs-block": "~0.8.1",
-    "ipfs-repo-migrations": "~0.1.0",
+    "interface-datastore": "^0.8.3",
+    "ipfs-repo-migrations": "^0.2.0",
+    "ipfs-utils": "^2.2.0",
+    "ipld-block": "^0.9.1",
```

Comment:
This is probably going to need a follow-up PR to js-ipfs at least that swaps ipfs-block for ipld-block

Reply:
yes! the following PR in js-ipfs to remove node globals will include that